packages/taco/integration-test/sign.test.ts

```diff
@@ -9,7 +15,7 @@ import { signUserOp } from '../src/sign';
 const RPC_PROVIDER_URL = 'https://ethereum-sepolia-rpc.publicnode.com';
 const DUMMY_ADDRESS = '0x742D35Cc6634C0532925A3b8D33c9c0E7B66C8E8';
 const DOMAIN = 'lynx';
-const RITUAL_ID = 1;
+const COHORT_ID = 1;
```

Comment:
out of the scope of this PR, but this is interesting. I have always thought that `ritual_id` is not a good name for the cohorts. I like cohort_id. Should we use this name from now on in all places?